docs/style/style-guide.md

```diff
@@ -28,16 +22,106 @@ Meeting this goal will require your careful consideration about when, where, and
 
 Javadocs within BFD will follow these conventions:
 
-- All methods and constructors of all scopes must have a javadoc
-  - Must have a description that explains what the method does, and any useful contextual/historical information that may help maintainability
-  - All method parameters should be documented with the ```@param``` tag, and a description of what the parameter is
-    - If ```null``` is explicitly disallowed for a parameter, it should be documented
-  - Method returns should be documented with the ```@returns``` tag and a description of what is returned (and how if multiple returns under various conditions)
-    - If the method can return ```null``` it should be noted and under what condition this may happen
-  - Method overrides or interface implementations should have a javadoc that is simply ```{@inheritDoc}``` unless the overridden method's functionality diverges enough to need additional documentation
-    - This includes ```toString```, ```hashCode```, etc.
-- All classes and interfaces of all scopes must have a javadoc 
-- All class-level fields of all scopes must have a javadoc
-    - Information about the field should be captured in the field javadoc, which can be referenced by the getter/setter to reduce duplication
-    - Reference other classes when it makes sense to reduce duplication (for instance, a class level field that represents a CcwCodebookVariable can just link to the CcwCodebookVariable in the documentation)
+**Classes**
+
+Classes must have a description; ideally this should be a high-level description of the class responsibility and/or purpose that is at least one full sentence ending in a period.
+
+```
```

Comment:
```suggestion
```java
```

Reply:
Good suggestion; applied this across the board.